src/main/java/org/generationcp/middleware/service/impl/study/StudyServiceImpl.java

```diff
@@ -214,7 +210,9 @@ public boolean hasMeasurementDataOnEnvironment(final int studyIdentifier, final
 
 	@Override
 	public boolean hasAdvancedOrCrossesList(final int studyId) {
-		return this.daoFactory.getGermplasmListDAO().hasAdvancedOrCrossesList(studyId);
+		final GermplasmStudySourceRequest searchParameters = new GermplasmStudySourceRequest();
```

Comment:
shall we rename the function? because the list does not exist anymore

Reply:
I think it's better to rename this to hasCrossesOrSelections.

Reply:
Done